loadtest/store/memstore/store.go

```diff
@@ -225,6 +235,18 @@ func (s *MemStore) SetClientConfig(config map[string]string) {
 	s.lock.Lock()
 	defer s.lock.Unlock()
 	s.clientConfig = config
+
+	// Populate FF
+	s.featureFlags = map[string]bool{}
+	for k, v := range s.clientConfig {
+		if ffKey := strings.TrimPrefix(k, "FeatureFlag"); len(ffKey) < len(k) {
+			v, err := strconv.ParseBool(v)
+			if err != nil {
+				continue
+			}
+			s.featureFlags[ffKey] = v
+		}
```

Comment:
Took me a second to understand the condition here. Could we do the following, even if there's some repetition?
```suggestion
		if strings.HasPrefix(k, "FeatureFlag") {
			v, err := strconv.ParseBool(v)
			if err != nil {
				continue
			}
			s.featureFlags[strings.TrimPrefix(k, "FeatureFlag")] = v
		}
```

Reply:
Funny that's how it was and then I simplified in https://github.com/mattermost/mattermost-load-test-ng/pull/631/commits/2762b4ebab76520ae41d272057f619b1039c1470

Reply:
Oh, I had not seen that :joy: I understand why you made the simplification, but my brain had to go through the non-simplified version to understand it.

Reply:
I am inclined to keep the simplification but will add a comment to explain what's going on.

Reply:
Sounds good, thanks!